src/frontend/js/components/CourseRunEnrollment/index.spec.tsx

```diff
@@ -128,6 +128,9 @@ describe('<CourseRunEnrollment />', () => {
 
     const button = screen.getByRole('button', { name: 'Enroll now' });
 
+    // eslint-disable-next-line no-console
+    console.error = jest.fn();
```

Comment:
We use a function to handle errors all over our frontend (https://github.com/openfun/richie/blob/master/src/frontend/js/utils/errors/handle.ts). According to the environment, it is in charge to send error to sentry or just display it through a `console.error`.

So to make this test more relevant, you have to mock our `handle` method then check it has been called.